appointment/upload-documents/uploadDocumentsController.js

```diff
@@ -202,6 +237,14 @@ define('uploadDocumentsController', [], function() {
 			});
 		};
 
+		$scope.validateConsent = (consentId) => {
```

Comment:
If we choose to keep it this way, is there a better way to do this? (Logically or with different operators)

Reply:
If we did choose to keep it this way, and I don't think we should, then you could just do `$scope.consentData.completeConsent = consentId != null`